server/src/universal/conf/delta-sharing-server.yaml.template

```diff
@@ -8,8 +8,10 @@ shares:
     tables:
     - name: "table1"
       location: "s3a://<bucket-name>/<the-table-path>"
+      precision: 0.1
```

Comment:
This is how the precision is added to the configuration. @cugni is it what we need?